### PR TITLE
Finish clause

### PR DESCRIPTION
**Pull request by hvub**
This PR documents the FINISH clause, cf. https://github.com/neo-technology/neo4j/pull/24360

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-898.surge.sh

New pages:
- [FINISH](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses/finish)

Updated pages:
- [Clauses](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 1]
```diff
@@ -15,6 +15,29 @@ New features are added to the language continuously, and occasionally, some feat
...
+
+a|
+label:syntax[]
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+FINISH
+----
+| Added the xref:clauses/finish.adoc[FINISH], which defines a query to have no result set.
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: "defines a query to have no result", while in line 48 of `modules/ROOT/pages/clauses/index.adoc` it says "defines that a query has no result set"

which one suits FINISH better? use in both cases :)

Comment (hvub) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: Both are meant to say the same. There is no "suits better". Question is which is better English? I guess the latter. Agreed?

Comment (rsill-neo4j) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: let's go with "defines a query to have no result"
i asked @fiquick for native speaker input (thanks! :) )

**Review by gem-neo4j — APPROVED** [review]
LGTM :)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-898.surge.sh

New pages:
- [FINISH](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses/finish)

Updated pages:
- [Clauses](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by rsill-neo4j — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-898.surge.sh

New pages:
- [FINISH](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses/finish)

Updated pages:
- [Clauses](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-898.surge.sh

New pages:
- [FINISH](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses/finish)

Updated pages:
- [Clauses](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/clauses/finish.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+The reading query
+
+.Query
+[source, cypher]
+----
+MATCH (p:Person)
+FINISH
+----
+
+successfully executes but has no result.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/finish.adoc]:
```suggestion

The following read query successfully executes but has no results

.Query
[source, cypher]
----
MATCH (p:Person)
FINISH
----
```

File: modules/ROOT/pages/clauses/finish.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+is equivalent to
+
+.Query
+[source, cypher]
+----
+CREATE (p:Person)
+----
+
+It has no result but creates one node with label `Person`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/finish.adoc]:
```suggestion
The following query has no result but creates one node with the label `Person`:

.Query
[source, cypher]
----
CREATE (p:Person)
FINISH
----

It is equivalent to the following query:

.Query
[source, cypher]
----
CREATE (p:Person)
----
```

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 4]
```diff
@@ -15,6 +15,29 @@ New features are added to the language continuously, and occasionally, some feat
...
+
+a|
+label:syntax[]
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+FINISH
+----
+| Added the xref:clauses/finish.adoc[FINISH], which defines a query to have no result.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
| New xref:clauses/finish.adoc[FINISH] clause, which defines a query to have no result.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-898.surge.sh

New pages:
- [FINISH](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses/finish)

Updated pages:
- [Clauses](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/clauses)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-898.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by JPryce-Aklundh — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**